MapboxMobileEvents/Categories/NSProcessInfo+SystemInfo.m

```diff
@@ -0,0 +1,65 @@
+#import "NSProcessInfo+SystemInfo.h"
+#include <sys/types.h>
+#include <sys/sysctl.h>
+#include <mach/machine.h>
+
+@implementation NSProcessInfo (SystemInfo)
+
++ (NSString*)stringForProcessorType:(NSInteger)type subtype:(NSInteger)subtype {
```

Comment:
We're in a public namespace so `mme_string…`

Reply:
Agreed. Updated